tests/test_preprints.py

```diff
@@ -39,99 +41,146 @@ class TestPreprintWorkflow:
+            submit_page.upload_project_selector.click()
+            submit_page.upload_project_help_text.here_then_gone()
+            submit_page.upload_project_selector_project.click()
+
+            submit_page.upload_select_file.click()
+            submit_page.upload_file_save_continue.click()
+
+            # Author Assertions section
+            # Note: We can't use the submit_page.save_author_assertions object here,
+            # because it is disabled and any time we use an object defined in
+            # pages/preprints.py it uses get_web_element() in the Locator class.
+            # Within get_web_element() the element_to_be_clickable method is used,
+            # and this method will always fail for disabled objects.  So in this
+            # instance we have to get the button object using the driver.find_element
+            # method while it is disabled.  After the button becomes enabled (i.e.
+            # after required data has been provided) then we can use the
+            # submit_page.save_author_assertions object to check the disabled
+            # property.  See implementation below.
+            assert driver.find_element(
+                By.CSS_SELECTOR, '[data-test-author-assertions-continue]'
+            ).get_property('disabled')
+            assert submit_page.public_data_input.absent()
+            submit_page.public_available_button.click()
+            assert submit_page.public_data_input.present()
+            submit_page.public_data_input.click()
+            submit_page.public_data_input.send_keys_deliberately('https://osf.io/')
+            # Need to scroll down since the Preregistration radio buttons are obscured
+            # by the Dev mode warning in test environments
+            submit_page.scroll_into_view(submit_page.preregistration_no_button.element)
+            assert submit_page.preregistration_input.absent()
+            submit_page.preregistration_no_button.click()
+            assert submit_page.preregistration_input.present()
+            submit_page.preregistration_input.click()
+            submit_page.preregistration_input.send_keys_deliberately('QA Testing')
+            # Save button is now enabled so we can use the object as defined in
+            # pages/preprints.py
+            assert submit_page.save_author_assertions.is_enabled()
+            submit_page.save_author_assertions.click()
+
+            submit_page.basics_license_dropdown.click()
+            # The order of the options in the license dropdown is not consistent across
+            # test environments. So we have to select by the actual text value instead
+            # of by relative position (i.e. 3rd option in listbox).
+            licenseSelect = Select(submit_page.basics_license_dropdown)
+            licenseSelect.select_by_visible_text('CC0 1.0 Universal')
+            # Need to scroll down since the Keyword/tags section is obscured by the Dev
```

Comment:
Nitpick: use `snake_case` for variable names.  `CamelCase` is for class names.

Reply:
Good catch.  I've corrected it.